### PR TITLE
Fixes[#5870] Card UX Improvement Mobile

### LINKED ISSUE #5870: Event card UX improvement mobile
### Description

On mobile, the event cards "depress" on click but nothing happens.  The user must click on the event title to navigate to the event.  This creates confusion in the user experience.

This can be addressed in 1 of 2 ways (to be discussed)
1. ~~remove the click event on the entire card and make it clear that the event title is clickable visually~~
2. adjust the click event so that on click of the event card, navigation to the event takes place

### Screenshots

See this video for the card click interactions

https://user-images.githubusercontent.com/97615019/230390611-ed5a55a8-3fe5-46d4-9230-4c86666ff2a5.mov

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Noothan-am**
## Fixes Issue

Closes #5870

## Changes proposed
Fixed it by wrapping whole card Under the `Link` Component.

## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6629"></a>

### CODE REVIEW

**Review by HassanTanveer — DISMISSED** [review]
LGTM

File: components/event/EventCard.js [hunk 1]
```diff
@@ -36,17 +36,23 @@ export default function EventCard({ event, username }) {
...
+            <Link
+              href={event.url}
+              key={event.url}
+              target="_blank"
+              rel="noreferrer"
+              className="text-decoration-line: none"
+            >
               <div className="flex justify-between">
-                <Link
+                <p
```
Comment (NiazMorshed2007) [inline components/event/EventCard.js:47]: I'm not sure why you used `href`, `key`, `target` ad `rel` to the `<p>`.
It needs to be removed,

**Review by NiazMorshed2007 — CHANGES_REQUESTED** [review]

File: components/event/EventCard.js (hunk 1 above)
Comment (Noothan-am) [reply components/event/EventCard.js]: i have removed it

**Comment by Noothan-am** [comment]
hey @NiazMorshed2007 i closed this and made a new pr #6671

**Pull request closed without merging**